Sources/Sentry/include/SentryCrashWrapper.h

```diff
@@ -1,8 +1,11 @@
 #import "SentryDefines.h"
+#import "SentryInternalDefines.h"
 #import <Foundation/Foundation.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef uint64_t bytes;
```

Comment:
since we have `SentryInternalDefines.h` now, do we still need this?

Reply:
Nope, good point